inception-project/inception
# #3112 - Allow additional user input when closing a document

**What's in the PR**
- New dialog for finishing documents which allows choosing between completing a document successfully or not successfully
- The new dialog also offers the option to leave a comment to the curator
- Added ability to see the comment to the curator via the workload management pages as well as on the curation page
- Reset the comment when resetting the document
- Include the comment in the webhook message on the annotation state change
- Updated webhook documentation
- Include the annotator comment in project export/import
- Include annotatorState and annotatorComment also in the event log

**How to test manually**
* Close document on annotation page
* Check if you see the comment on the dynamic / static workload management pages as well as on the curation page
* Export project, import it back in and check if the comments are still visible
* Configure webhook and see if the message is included
* Check that comment is cleared when document is reset

**Automatic testing**

**Documentation**
